src/components/MemoDetail.vue

```diff
@@ -0,0 +1,67 @@
+<template>
+  <div class="backdrop" @click.self='closeDetail'>
+    <div class="modal">
+      <h2>Memo Detail</h2>
+      <label>
+        <input placeholder="Title" v-model='this.memo.title'>
```

Comment:
`this` はなくても動いたりしませんかね...？
```suggestion
        <input placeholder="Title" v-model='memo.title'>
```

Reply:
動きました、削除します！